docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc

```diff
@@ -0,0 +1,85 @@
+= Create a MySQL service
+
+AppCat allows DevOps engineers to quickly provision a MySQL instance.
+
+== Create a MySQL service
+
+include::user:ROOT:partial$login-in-terminal.adoc[]
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to create an Object Storage bucket
```

Comment:
```suggestion
.Example to create a MySQL instance
```

Reply:
huh, how did that come in. sorry - overread. Thanks.